src/modules/navigator/rtl_params.c

```diff
@@ -159,3 +159,17 @@ PARAM_DEFINE_FLOAT(RTL_FLT_TIME, 15);
  * @group Return To Land
  */
 PARAM_DEFINE_INT32(RTL_PLD_MD, 0);
+
+/**
+ * Loiter radius for rtl descend
+ *
+ * Set the radius for loitering to a safe altitude for VTOL transition.
+ *
+ * @unit m
+ * @min 25
+ * @max 1000
+ * @decimal 1
+ * @increment 0.5
+ * @group Return Mode
+ */
+PARAM_DEFINE_FLOAT(RTL_LOITER_RAD, 50.0f);
```

Comment:
@dayjaby This is now the same default as NAV_LOITER_RAD. Does this really makes sense?

Reply:
I think not, but I would like some test flights first before setting it to a more sane value.